src/level-ext.adoc

```diff
@@ -0,0 +1,156 @@
+| 4       | ✔ | ✔ | ✔ | ✔  |    | _(3)_ |   |     | N/A | Reserved for `LVLBITS=2`
+| 5       | ✔ | ✔ | ✔ | ✔  |    | _(2)_ |   |     | N/A | Reserved for `LVLBITS=2`
+| 6       | ✔ | ✔ | ✔ | ✔  |    | 1     |   |     | N/A | Data & Cap RW (with store _local_, no <<el_perm>>)
+| 7       | ✔ | ✔ | ✔ | ✔  |    | 0     |   |     | N/A | Data & Cap RW (no store _local_, no <<el_perm>>)
+11+| *Quadrant 3: Capability data read/write*
+11+| bit[2] = write, bit[1:0] = store level. R and C implicitly granted.
+11+| _Reserved bits for future extensions must be 1 so they are implicitly granted_
+|Bits[4:3]| R | W | C | LM | EL | SL    | X | ASR | Mode^1^ |
+| 0-2   10+| reserved
+| 3       | ✔ |   | ✔ | ✔  | ✔  | N/A   |   |     | N/A | Data & Cap R0
+| 4       | ✔ | ✔ | ✔ | ✔  | ✔  | _(3)_ |   |     | N/A | Reserved for `LVLBITS=2`
+| 5       | ✔ | ✔ | ✔ | ✔  | ✔  | _(2)_ |   |     | N/A | Reserved for `LVLBITS=2`
+| 6       | ✔ | ✔ | ✔ | ✔  | ✔  | 1     |   |     | N/A | Data & Cap RW (with store _local_)
+| 7       | ✔ | ✔ | ✔ | ✔  | ✔  | 0     |   |     | N/A | Data & Cap RW (no store _local_)
+|==============================================================================
+
+[#section_cap_level_change]
+=== Changing capability levels and permissions
+While capability levels are conceptually a label on the capability rather than a permission granted by the capability, they are adjusted using the <<ACPERM>> instruction.
+This avoids the need for a dedicated instruction and allows reducing the level and removing <<el_perm>> in a single instruction.
+
+<<<
+
+[#section_cap_level_summary]
+=== Capability level summary table
+
+.{cheri_levels_ext_name} `LVLBITS=1` summary table for stored capabilities
+[#cap_level_store_summary,width="100%",options=header,align="center",cols="1,1,1,1,5"]
+|==============================================================================
+3+|Auth cap field | Data cap field | Notes
+|*W*|*C*|*SL*|*CL*|
+|1  | 1 | 1  | X  | Store data capability unmodified
+|1  | 1 | 0  | 1  | Store data capability unmodified
+|1  | 1 | 0  | 0  | Store data capability with tag cleared
+|==============================================================================
+
+NOTE: if W=0 or C=0 then SL is irrelevant
+
+.{cheri_levels_ext_name} `LVLBITS=1` summary table for loaded capabilities
+[#cap_level_load_summary,width="100%",options=header,align="center",cols="1,1,1,1,1,5"]
+|==============================================================================
+4+|Auth cap field | Data cap field |Notes
+|*R*|*C*|*EL*|*CL*| Tag            |
+|1  | 1 | 1  | X  | X              |Load data capability unmodified
+|1  | 1 | 0  | 1  | X              |Load data capability unmodified
+|1  | 1 | 0  | 0  | 1              |Load data capability with CL=0
+|1  | 1 | 0  | 0  | 0              |Load data capability unmodified
+|==============================================================================
```

Comment:
I think this table should be like this instead:
| CL | EL | | CL | EL |
|-|-|-|-|-|
| 0 | 0 | | 0 | 0 |
| 0 | 1 | | Unchanged | Unchanged |
| 1 | 0 | | Unchanged | 0 |
| 1 | 1 | | Unchanged | Unchanged |

Reply:
Fixed and cleaned up table.